src/puppetlabs/trapperkeeper/services/status/status_service.clj

```diff
@@ -0,0 +1,30 @@
+(ns puppetlabs.trapperkeeper.services.status.status-service
+  (:require [clojure.tools.logging :as log]
+            [compojure.core :as compojure]
+            [puppetlabs.trapperkeeper.core :refer [defservice]]
+            [puppetlabs.trapperkeeper.services :refer [service-context]]
+            [puppetlabs.trapperkeeper.services.status.status-core :as core]
+            [puppetlabs.trapperkeeper.services.webrouting.webrouting-service :as webrouting-service]))
+
+(defprotocol StatusService
+  (register-status [this service-name service-version status-version status-fn]
```

Comment:
I think everything you've said makes sense - sensible defaults and throwing an error in both those cases. Do you think that should be done as part of this ticket? (I think it does? but I could be convinced otherwise... I just don't really want to make another JIRA ticket.)